gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowStatusResource.java

```diff
@@ -186,7 +186,8 @@ static ExecutionStatus updatedFlowExecutionStatus(ExecutionStatus jobExecutionSt
     if (currentFlowExecutionStatus == ExecutionStatus.RUNNING ||
         jobExecutionStatus == ExecutionStatus.RUNNING ||
         jobExecutionStatus == ExecutionStatus.ORCHESTRATED ||
-        jobExecutionStatus == ExecutionStatus.COMPILED) {
+        jobExecutionStatus == ExecutionStatus.COMPILED ||
+        jobExecutionStatus == ExecutionStatus.PENDING) {
```

Comment:
This does not look right. A Pending job is not in running state yet, right?

Reply:
Actually this is correct. A pending *job* puts the *flow* in running state.
if this is the first job and it is pending, then flow will be *pending* determined by line 188